README.md

```diff
@@ -38,15 +38,16 @@ To get an idea of what this API looks like have a look at [triangle.py](https://
 pip install wgpu
 ```
 
-The library ships with Rust binaries for Windows, MacOS and Linux. If you want to use
+The wheels include the prebuild binaries. If you want to use
 a custom build instead, you can set the environment variable `WGPU_LIB_PATH`.
+You probably also want to install ``glwf`` (for desktop) and/or ``jupyter_rfb`` (for Jupyter).
```

Comment:
double backticks?

Reply:
I was editing rst right before editing this ;)